File: src/library/scala/runtime/java8/JFunction0$mcV$sp.java

/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */

package scala.runtime.java8;

@FunctionalInterface
public interface JFunction0$mcV$sp extends scala.Function0, java.io.Serializable {
    void apply$mcV$sp();

    default Object apply() { apply$mcV$sp(); return scala.runtime.BoxedUnit.UNIT; }
}

File: src/library/scala/runtime/java8/JFunction0$mcV$sp.scala
/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */
package scala.runtime.java8

@FunctionalInterface trait JFunction0$mcV$sp extends Function0[Any] with Serializable {
  def apply$mcV$sp(): Unit
  override def apply: Any = {
    apply$mcV$sp()
    scala.runtime.BoxedUnit.UNIT
  }
}